quickstart/known-issues.md

```diff
@@ -36,3 +36,9 @@ If this is not done, CSI socket paths won't match expected values and the Mayast
 
 When rebooting a node that runs applications mounting Mayastor volumes, this can take tens of minutes. The reason is the long default NVMe controller timeout \(`ctrl_loss_tmo`\). The solution is to follow the best k8s practices and cordon the node ensuring there aren't any application pods running on it before the reboot. Setting `ioTimeout` storage class parameter can be used to fine-tune the timeout.
 
+### Node restarts on scheduling an application 
+
+Deploying an application on a K8s environment which hosts Mayastor along with Prometheus exporter causes the application node to restart.
```

Comment:
```suggestion
Deploying an application pod on a worker node which hosts both Mayastor and Prometheus exporter causes that node to restart.
```

Reply:
Updated the text with suggested changes. -https://github.com/openebs/mayastor-docs/pull/33/commits/c72cbcc18b4ea4b119722bf3e814913e98ef9be4